### ISSUE TITLE
TypeError: install() got an unexpected keyword argument 'prune'

### ISSUE THREAD

**Issue by kalefranz** [conda/conda-env#252]
_From @komailrk on June 16, 2016 19:39_

While updating an environment using a yaml file (included below), got a Python trace back.
```
name: booboo_py35
dependencies:
- openssl=1.0.2h=1
- pip=8.1.2=py35_0
- python=3.5.1=0
- readline=6.2=2
- setuptools=23.0.0=py35_0
- sqlite=3.13.0=0
- tk=8.5.18=0
- wheel=0.29.0=py35_0
- xz=5.0.5=1
- zlib=1.2.8=3
- pip:
  - pexpect==4.1.0
  - ptyprocess==0.5.1
prefix: /localdata/anaconda/envs/booboo_py35
```

```
((py35)) bash-4.1$ conda env update -f /home/booboo/conda_py35.yaml -n py35
Fetching package metadata .......
Solving package specifications .............
An unexpected error has occurred, please consider sending the
following traceback to the conda GitHub issue tracker at:

    https://github.com/conda/conda/issues

Include the output of the command 'conda info' in your report.


Traceback (most recent call last):
  File "/opt/anaconda/booboo/bin/conda-env", line 6, in <module>
    sys.exit(conda_env.cli.main.main())
  File "/opt/anaconda/booboo/lib/python2.7/site-packages/conda_env/cli/main.py", line 68, in main
    return args_func(args, parser)
  File "/opt/anaconda/booboo/lib/python2.7/site-packages/conda/cli/main.py", line 127, in args_func
    args.func(args, p)
  File "/opt/anaconda/booboo/lib/python2.7/site-packages/conda_env/cli/main_update.py", line 118, in execute
    installer.install(prefix, specs, args, env, prune=args.prune)
TypeError: install() got an unexpected keyword argument 'prune'
```

_Copied from original issue: conda/conda#2724_

**Comment by edparcell** [comment]
I encountered the same problem, affecting 2.5.0. It was introduced 10/29/2015 in e01ba6f3894be107fc0b14f5eb7326b012a51cbd, and was fixed yesterday in c050ba831b0a2403f92f6ba85b8ac680e5a0d7ae.

I was able to roll back in some cases with `conda install conda-env=2.4.5`. In others, I had to reinstall anaconda from scratch. Packaging tool problems are particularly troublesome because you typically need to use your packaging tools to resolve them.
